### DISCUSSION TITLE
Infomation error when merging subtitles

### DISCUSSION THREAD

**Discussion by ard-av** [SubtitleEdit/subtitleedit#6410] [Q&A]
![Screenshot_1](https://user-images.githubusercontent.com/114281218/201464715-8033e82a-310b-452b-bfbd-eabbbf7f662c.png)
In "end" column Shows the beginning of the last range, not its end

The example should not be
02:04:59,000 is 2:04:59.986, right?

**Comment by niksedk** [comment]
I think the source data is just too poor...

Is this from an actual subtitle file? Could you attach it?

**Reply by ard-av** [reply]
![Screenshot_1](https://user-images.githubusercontent.com/114281218/202192475-b4feeb5b-56ea-40bf-b406-af6f4a289d01.png)
![Screenshot_2](https://user-images.githubusercontent.com/114281218/202192484-48d0c2ed-0fd5-4af7-aeb6-aaffba9553fc.png)
[test.zip](https://github.com/SubtitleEdit/subtitleedit/files/10022052/test.zip)

**Reply by ard-av** [reply]
The result of the union is correct. Shows the duration incorrectly (the beginning of the last block, not the end of the last block)

**Reply by ard-av** [reply]
In the example, subtitle segments are 2 hours long, but information is displayed that the segments are one hour long

**Comment by ard-av** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/114281218/203242894-41a24d53-3c7a-4003-89ca-865c4e6a769e.png)
![Screenshot_2](https://user-images.githubusercontent.com/114281218/203242898-e5caed0b-e268-4488-b679-337b6dee71bc.png)
Greetings. Version 3.6.9. Situation not fixed
